services/rolemanager.md

```diff
@@ -9,21 +9,25 @@ Assign roles to devices on the JACDAC bus.
     ro all_roles_allocated: bool @ 0x181
 
 Indicates if all required roles have been allocated to devices.
+TODO is this needed?
```

Comment:
this is a cheap way to ask the role manager some roles need to be allocated. Otherwise, one has to setup a pipe for required_roles, etc...
We could replace it with a status code in the status register.